introduction.md

```diff
@@ -0,0 +1 @@
+Hi I am Chris and I think this is really stupid. Please allow a very short file. Also you have a typo in the word "sentence"
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.